.github/workflows/lint-js.yml

```diff
@@ -20,9 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
```

Comment:
We could probably set this to `lts/*` to avoid having to update it again in the future.